src/applications/ivc-champva/10-7959f-1/config/submitTransformer.js

```diff
@@ -10,13 +10,15 @@ export default function transformForSubmit(formConfig, form) {
     veteran: {
       date_of_birth: transformedData.veteranDateOfBirth,
       full_name: transformedData?.veteranFullName,
-      physical_address: transformedData.physicalAddress || {
-        country: 'NA',
-        street: 'NA',
-        city: 'NA',
-        state: 'NA',
-        postalCode: 'NA',
-      },
+      physical_address: transformedData.sameMailingAddress
+        ? transformedData.veteranAddress
+        : transformedData.physicalAddress || {
+            country: 'NA',
+            street: 'NA',
+            city: 'NA',
+            state: 'NA',
+            postalCode: 'NA',
+          },
```

Comment:
Now, when addresses have been designated as the same, we show the same address text in both spots on the form rather than using NAs by default. Only if the addresses are different AND we don't actually have a `physicalAddress` value do we resort to NAs